dottle

```diff
@@ -1,106 +1,85 @@
 #!/bin/sh
 
-# DEBUG=$(printf "${DEBUG}" | tr '[:upper:]' '[:lower:]')
-
+DEBUG=false
```

Comment:
This will always disable debug... before it was disabled by default (DEBUG variable was empty) and if you set it to anything (by calling dottle like `DEBUG=true ./dottle install`) it would activate the debug mode.